### PR TITLE
create signup step1 personal information component

### PR DESCRIPTION
**Pull request by abdulrahman-2020**
Relates #91
create signup step 1 component
check email exsist and send message from back

### CODE REVIEW

File: client/src/Pages/SignUp.tsx [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 
+import PersonalForm from '../Components/PersonalInformaion';
```
Comment (devstm) [inline client/src/Pages/SignUp.tsx:4]: export PersonalInformaion to Component/index.tsx then import it from there

File: client/src/Pages/SignUp.tsx [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+
 function SignUp() {
   return (
     <div className="container">
-      {useTranslation().t('signUp-greeting')}
+      {useTranslation().t('crate-account')}
```
Comment (devstm) [inline client/src/Pages/SignUp.tsx:9]: destruct  t from useTranslation() becaouse we use more than once

File: client/src/localization/local/ar.js [hunk 3]
```diff
@@ -58,6 +58,21 @@ const ar = {
   'successfully-delete': 'لقد تم حذف هذا العمل بنجاح',
   'successfully-delete-load': 'يتم الان الحذف ...',
   'error-delete-message': 'لقد حدث خطأ ما. حاول في وقت لاحق',
+
+  'First Name': 'الاسم الاول',
+  'Last Name': 'الاسم الاخير',
+  Email: 'البريد الالكتروني',
```
Comment (devstm) [inline client/src/localization/local/ar.js:64]: abdullah use 
  'login-email': 'البريد الالكتروني',
so we can use it instead of write another

**Review by devstm — CHANGES_REQUESTED** [review]

**Review by AbdallahGot — APPROVED** [review]

File: server/controllers/signUp/index.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      where: { email },
+    });
+    // console.log(user.email);
+    console.log(user);
+    if (user) {
+      // if (user.email === email) {
+      throw customError('Email already exists ...', 400);
+      // }
+    }
+    res.json({ msg: 'Sign up successful', status: 200 });
```
Comment (LinaYahya) [inline server/controllers/signUp/index.js:21]:
```suggestion
    res.json({ msg: 'email valid, status: 200 });
```

File: client/src/Components/PersonalInformaion/index.tsx [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+          label={t('login-email')}
+          name="email"
+          hasFeedback={hasFeedBack}
+          validateStatus={error ? 'error' : 'success'}
+          help={error}
+          rules={[{ required: true, message: t('required-email') }, { type: 'email', message: t('invalid-email') }]}
+        >
+          <Input
+            onBlur={register}
+            onChange={handleChange}
```
Comment (LinaYahya) [inline client/src/Components/PersonalInformaion/index.tsx:88]:
```suggestion
```

File: client/src/Components/PersonalInformaion/index.tsx [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+  const [error, setError] = useState('');
+  const [hasFeedBack, setHasFeedBack] = useState(false);
+  const handleChange = ({ target: { name, value } }:any) => {
+    setUserInfo({ ...userInfo, [name]: value });
+  };
+  const register = async () => {
+    setError('');
+    setHasFeedBack(true);
+    try {
+      const { data } = await axios.post('/api/v1/signUp', userInfo);
```
Comment (LinaYahya) [inline client/src/Components/PersonalInformaion/index.tsx:28]: change endpoint to `/checkemail`
and only send email

**Review by LinaYahya — CHANGES_REQUESTED** [review]

**Review by LinaYahya — APPROVED** [review]

**Review by devstm — APPROVED** [review]

**Review by RandSohail — APPROVED** [review]

**Pull request merged**